rn/Teacher/ios/TeacherUITests/Login/LoginStartTests.swift

```diff
@@ -0,0 +1,28 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2018-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+@testable import Core
+import TestsFoundation
+import XCTest
+class DemoLoginTest: TeacherUITestCase {
```

Comment:
Usually we try to match the class name to the file name. So this should either be `class LoginStartTests` or be in `DemoLoginTests.swift`